src/fides/api/alembic/migrations/versions/5a4859f74832_add_last_ran_at_to_connectionconfig.py

```diff
@@ -0,0 +1,31 @@
+"""Add last_ran_at to ConnectionConfig
+
+Revision ID: 5a4859f74832
+Revises: c9a22b284afa
+Create Date: 2024-10-22 14:58:09.174708
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "5a4859f74832"
+down_revision = "c9a22b284afa"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "connectionconfig",
+        sa.Column("last_ran_at", sa.DateTime(timezone=True), nullable=True),
```

Comment:
Sorry to be annoying, how do we feel about `last_run_time`? Non blocking.

Reply:
sure. I'll change it to `last_run_timestamp` so it's consistent with `last_test_timestamp`